test/BlendSwap.js

```diff
@@ -296,7 +240,7 @@ describe('BlendSwap', async function() {
             await time.increaseTo(releaseTime)
             await expectRevert(
                 ctx.swap.claimRefund(dummySecretHash, {from: bob}),
-                "Wrong status"
+                "Lock does not exists"
             )
         })
 
```

Comment:
Also check that `swaps` mapping doesn't hold swap anymore after successful `claimRefund`